internal/controllers/topology/cluster/patches/external/external_patch_generator.go

```diff
@@ -48,6 +48,10 @@ func (e externalPatchGenerator) Generate(ctx context.Context, forObject client.O
 	if !feature.Gates.Enabled(feature.RuntimeSDK) {
 		return nil, errors.Errorf("can not use external patch %q if RuntimeSDK feature flag is disabled", *e.patch.External.GenerateExtension)
 	}
+
+	// Set the settings defined in external patch definition on the request object.
```

Comment:
Let's add a note that those settings applies on top of settings defined on the ExtensionConfig